components/studio/bin/hab-studio.sh

```diff
@@ -1220,7 +1087,7 @@ unset PATH
 
 # The root path of the Habitat file system. If the `$HAB_ROOT_PATH` environment
 # variable is set, this value is overridden, otherwise it is set to its default
-: ${HAB_ROOT_PATH:=/hab}
+: "${HAB_ROOT_PATH:=/hab}"
 # The root path containing all locally installed packages
 HAB_PKG_PATH=$HAB_ROOT_PATH/pkgs
```

Comment:
This is the only remaining linter error:
[`HAB_PKG_PATH` appears unused. Verify it or export it.](https://github.com/koalaman/shellcheck/wiki/SC2034)

I'm hesitant to remove it, but I don't see how it's used at all. Thoughts?

Reply:
That variable is used inside of a `plan.sh` @baumanj see https://www.habitat.sh/docs/reference/#plan-variables

Reply:
Yeah, but I think in that instance it's set here in [hab-plan-build.sh](https://github.com/habitat-sh/habitat/blob/master/components/plan-build/bin/hab-plan-build.sh#L351). As best I can tell, the only environment variables that pass from `hab-studio.sh` into subcommands are those that are included by [`chroot_env`](https://github.com/habitat-sh/habitat/blob/master/components/studio/bin/hab-studio.sh#L928), but `HAB_PKG_PATH` is not among those. Am I missing something, @mwrock ?

Reply:
ugh. apologies. I was thinking of plan build. not studio.

Reply:
Hey, thanks for looking! I'm like 95% sure it should be removed, but I'll wait for @fnichol and/or @elliott-davis to weigh in.

Reply:
Yep, it's used in the `baseimage` and `bare` Studio type implementations (found this with a grep through the codebase). I'd add an ignore for that one.